src/Data/Blob.hs

```diff
@@ -44,12 +45,12 @@ data File = File
   , fileLanguage :: Language
   } deriving (Show, Eq, Generic)
 
--- | Prefer 'fileForRelPath' if at all possible.
+-- | Prefer 'fileForTypedPath' if at all possible.
 fileForPath :: FilePath  -> File
```

Comment:
We can’t _quite_ kill this yet because there are still some CLI actions that create a `File` even though the `FilePath` they wrap is a directory. This is a wart, yes, but the changes required fall outside of the purview of this patch, so I’m leaving it for now.